code/game/data_huds.dm

```diff
@@ -620,30 +620,41 @@
 /datum/atom_hud/order
 	hud_icons = list(ORDER_HUD)
 
+///Updates aura hud icons
 /mob/living/carbon/human/proc/hud_set_order()
 	var/image/holder = hud_list[ORDER_HUD]
-	holder.icon_state = ""
-	if(stat != DEAD)
-		var/tempname = ""
-		if(mobility_aura)
-			tempname += "move"
-		if(protection_aura)
-			tempname += "hold"
-		if(marksman_aura)
-			tempname += "focus"
-		if(tempname)
-			holder.icon = 'icons/mob/hud/aura.dmi'
-			holder.icon_state = "[tempname]"
-
-	hud_list[ORDER_HUD] = holder
+	holder.overlays.Cut()
```

Comment:
I know it's not your fault but this should really be done properly (in another PR probably)

Reply:
Feel free to TM yourself